src/main/java/frc/robot/commands/groups/ScoreLow.java

```diff
@@ -4,19 +4,21 @@
 import edu.wpi.first.wpilibj2.command.WaitCommand;
 import frc.robot.Constants;
 import frc.robot.commands.floorArm.FloorMoveArmToPostion;
+import frc.robot.commands.intake.armIntake.DropCone;
 import frc.robot.commands.intake.armIntake.DropCube;
 import frc.robot.subsystems.DrivetrainSubsystem;
 import frc.robot.subsystems.FloorArmSubsystem;
 import frc.robot.subsystems.Intake;
 
-public class ScoreCubeLow extends SequentialCommandGroup {
+public class ScoreLow extends SequentialCommandGroup {
 
-  public ScoreCubeLow(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, Intake intake) {
+  public ScoreLow(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, Intake intake) {
     addCommands(
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_SHOOTING_POSITION),
       new WaitCommand(0.5),
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT_POSITION),
-      new DropCube(intake).withTimeout(0.5)
+      new DropCube(intake).withTimeout(0.5),
+      new DropCone(intake).withTimeout(0.5)
```

Comment:
I have some concern for the cone case of I taking too far if trying to drop the cube. We might eventually need two commands